assets/component-localization-form.css

```diff
@@ -222,7 +222,7 @@ noscript .localization-selector.link {
 .desktop-localization-wrapper {
   display: flex;
   flex-wrap: wrap;
-  justify-content: flex-end;
+  justify-content: space-around;
```

Comment:
I think we should keep `flex-end` here. So we consistently place the selectors to the far right. Otherwise we end up having this space from `space-around`: 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/05-11-afs40-39hi7.png)
</details>

Which might make their positioning inconsistent. If we want some spacing between the two, you could do something like this: 
```css
localization-form + localization-form {
  margin-right: XXrem;
}
```

When the selectors wraps, it also keeps them centered: 
<details> 
<summary>Screenshot</summary>

![](https://screenshot.click/05-14-mz4w5-uewrg.png)
</details>